module/reports/helper.js

```diff
@@ -7,736 +7,879 @@
 
 // Dependencies
 
-const userProjectsHelper = require(MODULES_BASE_PATH + "/userProjects/helper");
```

Comment:
@MallanagoudaB What is the change in this file or is it just formatting?

Reply:
formatting

Reply:
i have made changes for entity API